src/unity.yml

```
@@ -17,6 +17,8 @@ paths:
     $ref: './unity/paths/accounts_accountId.yml'
   '/accounts/{accountId}/orgs':
     $ref: './unity/paths/accounts_accountId_orgs.yml'
+  '/accounts/{accountId}/orgs/default':
+    $ref: './unity/paths/accounts_org_default.yml'
```

I think the file name should be `accounts_accountId_orgs_default` to match the naming standards, just basing it on the others. Just let me know if I'm wrong.

```suggestion
    $ref: './unity/paths/accounts_accountId_orgs_default.yml'
```